src/extension/launchArgs.ts

```diff
@@ -25,6 +25,7 @@ export interface IAndroidRunOptions extends ILaunchArgs {
     variant?: string;
     logCatArguments?: any;
     debugLaunchActivity?: string;
+    isDirect?: boolean;
```

Comment:
Can we move this property to `ILaunchArgs`?

Reply:
Yes, It would be better to move this parameter to `ILaunchArgs`.